src/main/java/com/itschool/eventmanagment/models/dtos/EventDTO.java

```diff
@@ -13,12 +13,11 @@
 public class EventDTO {
     private Long id;
     private String name;
-    //private LocalDateTime eventDateTime;
     private String location;
     @JsonDeserialize(using = LocalDateTimeDeserializer.class)
     @JsonSerialize(using = LocalDateTimeSerializer.class)
     private LocalDateTime eventDateTime;
     private Long userId;
```

Comment:
If we have a property called `maxParticipants`, should we also have `participantId`? Maybe we can try to merge somehow the naming of these two properties so we can have a better intent with the properties.

User in this DTO doesn't say much. User of what? User of an event? An event can have participants, I guess. Those are some questions that I asked myself when I saw this `userId` property.

Also, this id property can be among the first properties that we have; we can consider having it under the first property of this class (id).

Reply:
@AlexandruIliescu for the first point I didn't get it, but I can keep this in mind and  ask
"If we have a property called maxParticipants, should we also have participantId? Maybe we can try to merge somehow the naming of these two properties so we can have a better intent with the properties."

This part with UserDTO, also is a bit confusing that was in the specification to use a user which create a event. 

Regarding the id it's was add it by Denisa but I've move in the top of the list.